its/src/test/java/com/sonar/it/csharp/AnalysisWarningsTest.java

```diff
@@ -20,19 +20,17 @@
 package com.sonar.it.csharp;
 
 import com.sonar.it.shared.TestUtils;
-import com.sonar.it.shared.Tests;
 import com.sonar.orchestrator.build.BuildResult;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 import org.sonarqube.ws.Ce;
 
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-
-import static com.sonar.it.shared.Tests.ORCHESTRATOR;
+import static com.sonar.it.csharp.Tests.ORCHESTRATOR;
```

Comment:
This is the actual change - from `shared` to `csharp`. The rest is autoformatting noise

Reply:
Nice catch!

Reply:
can we do something to reduce the visibility and avoid having this issue in the future?

Reply:
I don't think so, when I was doing it, I checked `@ClassRule` docs and it says

>A field must be public, static, and a subtype of [TestRule](https://junit.org/junit4/javadoc/4.12/org/junit/rules/TestRule.html).